src/main/java/seedu/address/logic/commands/AddMeetingCommand.java

```diff
@@ -59,7 +59,8 @@ public CommandResult execute(Model model) throws CommandException {
                 oldClient.getPhone(),
                 oldClient.getEmail(),
                 oldClient.getAddress(),
-                oldClient.getTags());
+                oldClient.getTags(),
+                oldClient.getProducts());
```

Comment:
Are we keeping tags and adding product? I thought we going to refactor tags to product?

Reply:
agree, we should refactor tags to products